### PR TITLE
create pdf out of the documentation

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by whummer — APPROVED** [review]
Looks great, kudos for putting this together so quickly @HarshCasper ! 🚀 

Added a few tiny comments, nothing that should block the merge..

Small note: seems like the border on the PDF pages is pretty small, hence the content sits very much at the top of the page (at least in my PDF reader). No need to change it for now though, could be done in a future iteration. 👍 

Example 1:
image

Example 2:
image

File: scripts/generate_pdf_output.py
Comment (HarshCasper) [reply scripts/generate_pdf_output.py:136]: Yeah, will follow up on the other PR 👍

File: scripts/pdf_list.py
Comment (HarshCasper) [reply scripts/pdf_list.py:130]: Seems like a bug. Will fix later in follow-up PR

File: scripts/generate_pdf_output.py
Comment (HarshCasper) [reply scripts/generate_pdf_output.py:122]: Yeah, this is required to setup all the PDF files before processing and merged into a final PDF

**Pull request merged**
